inc/namespace.php

```diff
@@ -135,13 +135,48 @@ function get_json_file_contents_as_array( $file ) : array {
 
 	if ( json_last_error() !== JSON_ERROR_NONE ) {
 		// phpcs:ignore
-		trigger_error( json_last_error_msg(), E_USER_WARNING );
+		trigger_error( 'composer.json could not be parsed\n' . json_last_error_msg(), E_USER_WARNING );
```

Comment:
Worth noting that the newline character `\n` will not be parsed as a newline unless it's inside double quotes

Reply:
We should avoid newlines in error messages anyway (as they can make parsing harder later), suggest:
```suggestion
		trigger_error( sprintf( 'composer.json could not be parsed: %s', json_last_error_msg() ), E_USER_WARNING );
```